scripts/globals/spells/enfeebling_spell.lua

```diff
@@ -416,6 +416,7 @@ xi.spells.enfeebling.useEnfeeblingSpell = function(caster, target, spell)
         if
             caster:isPC() and
             target:isMob() and
+            xi.settings.main.ENABLE_IMMUNOBREAK and
```

Comment:
I'd probably move this setting to be the first thing thats checked in this block

Reply:
Can do

Reply:
Done